cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/framework/image/NV21Image.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("deprecation")
+/* RenderScript is deprecated, but alternatives are not yet well supported */
```

Comment:
Same as Camera, can you explain what features are required to move away from the deprecated class?

Reply:
According to the [android developer blog](https://android-developers.googleblog.com/2021/04/android-gpu-compute-going-forward.html), We should be maintaining two codepaths going forward to keep taking advantage of the device GPU for image processing. Native vulkan is only supported on API 29+.

It seems like more investigation is needed here to determine what the impact of using the CPU only [renderscript-intrinsics-replacement-toolkit](https://github.com/android/renderscript-intrinsics-replacement-toolkit/blob/main/renderscript-toolkit/src/main/java/com/google/android/renderscript/Toolkit.kt) vs. native vulkan is.

I'll add this to the comment.